sunpy/map/sources/soho.py

```diff
@@ -96,7 +95,71 @@ def _supported_observer_coordinates(self):
     @classmethod
     def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an EIT image"""
-        return header.get('instrume') == 'EIT' or header.get('telescop') == 'Extreme-ultraviolet Imaging Telescope (EIT)'
+        return (header.get('instrume') == 'EIT' or header.get('telescop') == 'Extreme-ultraviolet Imaging Telescope (EIT)'  and header.get('level') is None)
+
+
+class EITL1Map(EITMap):
+    """
+    SOHO EIT L1 Image Map.
+
+    See the docstring of `EITMap` for information on the EIT instrument.
+
+    The EIT L1 data is a level 1 data product with very different headers to the normal/older EIT data.
+    This has required a new class to be created to handle the differences.
+
+    References
+    ----------
+    * `SOHO Mission Page <https://sohowww.nascom.nasa.gov/>`__
+    * `SOHO EIT Instrument Page <https://umbra.nascom.nasa.gov/eit/>`__
+    * `SOHO EIT User Guide <https://umbra.nascom.nasa.gov/eit/eit_guide/>`__
+    """
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+        self._nickname = self.detector
+        self.plot_settings['cmap'] = f"sohoeit{str(int(self.wavelength.to('angstrom').value))}"
+        self.plot_settings['norm'] = ImageNormalize(
+            stretch=source_stretch(self.meta, AsinhStretch(0.0001)), clip=False)
+
+    @property
+    def date(self):
+        # Old EIT data has date-obs in format of dd-JAN-yy so we use date_obs where available
+        return self._get_date('date_obs') or super().date
+
+    @property
+    def processing_level(self):
+        """
+        Returns the FITS processing level if present.
+        """
+        return self.meta.get('LEVEL', None)
+
+    @property
+    def detector(self):
+        return "EIT"
+
+    @property
+    def observatory(self):
```

Comment:
And this

Reply:
This gives a different answer for the L1 files.